bookstore/src/main/webapp/WEB-INF/layouts/classic.jsp

```diff
@@ -0,0 +1,57 @@
+<%@ taglib uri="http://tiles.apache.org/tags-tiles" prefix="tiles"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page isELIgnored="false"%>
+<%@ page contentType="text/html; charset=UTF-8" %>
+
+<html>
+<head>
+	<title><tiles:getAsString name="title" /></title>
+
+	<link href="<c:url value="/css/bootstrap-4.4.1/bootstrap.min.css" />" rel="stylesheet">
+	<link href="<c:url value="/css/bootstrap-4.4.1/bootstrap-glyphicons.css" />" rel="stylesheet">
+	<link href="<c:url value="/css/jquery-ui-1.12.1.css" />" rel="stylesheet">
+	<link href="<c:url value="https://fonts.googleapis.com/css?family=Muli:300,400,500,600,700,800,900&display=swap" />" rel="stylesheet">
+    <link href="<c:url value="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />" rel="stylesheet">
```

Comment:
@danhtrancong 
Should not use the online css.
Download the css https://fonts.googleapis.com/css and "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css use as local file.